media-gfx/blender/blender-2.79b.recipe

```diff
@@ -169,26 +169,30 @@ BUILD()
 {
 	PORT_SUBFOLDER=`echo $portVersion | sed 's/.$//'`
 
-	# OpenCOLLADA workaround
-	COLLADA_HEADERS="`finddir B_SYSTEM_HEADERS_DIRECTORY`/opencollada"
-	COLLADA_INC_FLAGS="-I$COLLADA_HEADERS/COLLADABaseUtils \
-		-I$COLLADA_HEADERS/COLLADAFramework \
-		-I$COLLADA_HEADERS/COLLADASaxFrameworkLoader \
-		-I$COLLADA_HEADERS/COLLADAStreamWriter \
-		-I$COLLADA_HEADERS/GeneratedSaxParser"
+	INCLUDE_DIR="`finddir B_SYSTEM_HEADERS_DIRECTORY`${secondaryArchSubDir}"
 
 	# Haiku specific settings
 	BLENDER_PLATFORM_SPECIFIC="-DCMAKE_INSTALL_PREFIX:PATH=$appsDir/Blender \
 		-DPYTHON_VERSION=${PYTHON_VERSION} \
 		-DPYTHON_INCLUDE_DIR=`finddir B_SYSTEM_HEADERS_DIRECTORY`/python${PYTHON_VERSION}m/ \
-		-DPYTHON_LIBRARY=`finddir B_SYSTEM_LIB_DIRECTORY`/libpython${PYTHON_VERSION}m.so.1.0 \
+		-DPYTHON_LIBRARY=`finddir B_SYSTEM_LIB_DIRECTORY`${secondaryArchSubDir}/libpython${PYTHON_VERSION}m.so.1.0 \
 		-DPYTHON_EXECUTABLE=/bin/python${PYTHON_VERSION} \
 		-DPYTHON_NUMPY_PATH=`finddir B_SYSTEM_LIB_DIRECTORY`/python${PYTHON_VERSION}/vendor-packages \
 		-DPYTHON_REQUESTS_PATH=`finddir B_SYSTEM_LIB_DIRECTORY`/python${PYTHON_VERSION}/vendor-packages \
-		-DOPENJPEG_INCLUDE_DIR=`finddir B_SYSTEM_HEADERS_DIRECTORY` \
-		-DOPENJPEG_LIBRARY=`finddir B_SYSTEM_LIB_DIRECTORY` \
-		-DZLIB_LIBRARY_RELEASE=`finddir B_SYSTEM_LIB_DIRECTORY`/libz.so.1 \
+		-DOPENCOLLADA_COLLADABASEUTILS_INCLUDE_DIR=${INCLUDE_DIR}/opencollada/COLLADABaseUtils \
+		-DOPENCOLLADA_COLLADAFRAMEWORK_INCLUDE_DIR=${INCLUDE_DIR}/opencollada/COLLADAFramework \
+		-DOPENCOLLADA_COLLADASAXFRAMEWORKLOADER_INCLUDE_DIR=${INCLUDE_DIR}/opencollada/COLLADASaxFrameworkLoader \
+		-DOPENCOLLADA_COLLADASTREAMWRITER_INCLUDE_DIR=${INCLUDE_DIR}/opencollada/COLLADAStreamWriter \
+		-DOPENCOLLADA_GENERATEDSAXPARSER_INCLUDE_DIR=${INCLUDE_DIR}/opencollada/GeneratedSaxParser \
+		-DOPENJPEG_INCLUDE_DIR=${INCLUDE_DIR} \
```

Comment:
Adding a check for opensubdiv here works on detection:
```diff
+		-DOPENSUBDIV_INCLUDE_DIR=`finddir B_SYSTEM_HEADERS_DIRECTORY`/opensubdiv
```

Reply:
This patch already has this:
```
-DOPENSUBDIV_INCLUDE_DIR=${INCLUDE_DIR} \
```
Detection on 32 bit is confirmed.

Reply:
Still trying to find the correct line to use :)

Reply:
What do you mean? Does solution in this patch not work for you?

Reply:
Grabbing your latest changes without this to check